src/posix.mak

```diff
@@ -351,7 +351,7 @@ $G/backend.a: $(G_OBJS)
 $G/dmd_frontend: $(FRONT_SRCS) $D/gluelayer.d $(ROOT_SRCS) $G/newdelete.o $(STRING_IMPORT_FILES) $(HOST_DMD_PATH)
 	CC=$(HOST_CXX) $(HOST_DMD_RUN) -of$@ $(MODEL_FLAG) -vtls -J$G -J../res -L-lstdc++ $(DFLAGS) $(filter-out $(STRING_IMPORT_FILES) $(HOST_DMD_PATH),$^) -version=NoBackend
 
-dmd: $G/dmd
+dmd: $G/dmd $G/dmd.conf
```

Comment:
This might be the biggest proposed change but I couldn't see any reason why we shouldn't auto-generate the `dmd.conf` into `generated` during a normal build.